ADALiOS/ADALiOS/ADLogger.m

```diff
@@ -144,12 +126,12 @@ + (void)log:(ADAL_LOG_LEVEL)logLevel
             if (s_NSLogging)
             {
                 //NSLog is documented as thread-safe:
-                NSLog([self formatStringPerLevel:logLevel], [dateFormatter stringFromDate:[NSDate date]], [[ADLogger getCorrelationId] UUIDString], message, info, errorCode);
+                NSLog([self formatStringPerLevel:logLevel], [dateFormatter stringFromDate:[NSDate date]], correlationId?[correlationId UUIDString]:@"", message, info, errorCode);
             }
             
             if (s_LogCallback)
             {
-                s_LogCallback(logLevel, [NSString stringWithFormat:@"ADALiOS [%@ - %@] %@", [dateFormatter stringFromDate:[NSDate date]], [[ADLogger getCorrelationId] UUIDString], message], info, errorCode);
+                s_LogCallback(logLevel, [NSString stringWithFormat:@"ADALiOS [%@ - %@] %@", [dateFormatter stringFromDate:[NSDate date]], correlationId?[correlationId UUIDString]:@"", message], info, errorCode);
```

Comment:
I think I would use a different format string based on whether or not a correlationId was passed in.